### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by dijonkitchen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ render() {
 }
 ```
 
-The `Loop` component uses `context` to pass a subscrible game tick down your component tree. The `Stage` component does the same with game scale.
+The `Loop` component uses `context` to pass a subscribe game tick down your component tree. The `Stage` component does the same with game scale.
```
Comment (triptych) [inline README.md]: I don't think that's right. I think it should be "subscribable". Not subscribe.

Comment (dijonkitchen) [inline README.md]: Fixed!

**Comment by smartandsexy** [comment]
Thanks

**Comment by dawsonbotsford** [comment]
lgtm

**Pull request merged**